.vsts-pipelines/variables/docker-images.yml

```diff
@@ -1,3 +1,3 @@
 variables:
-  imageNames.imageBuilder.linux: mcr.microsoft.com/dotnet-buildtools/image-builder:debian-20190326143958
-  imageNames.imageBuilder.windows: mcr.microsoft.com/dotnet-buildtools/image-builder:nanoserver-20190326144001
+  imageNames.imageBuilder.linux: mcr.microsoft.com/dotnet-buildtools/image-builder:debian-20190410133145
```

Comment:
Would you mind updating this to the latest `20190420013239` to get the publish manifest optimization?